.env.defaults

```
@@ -4,10 +4,10 @@
 # into version control.
 
 # schema.prisma defaults
-DATABASE_URL=file:./dev.db
+DATABASE_URL=postgresql://gitpod@localhost/dev
 
 # location of the test database for api service scenarios (defaults to ./.redwood/test.db if not set)
-# TEST_DATABASE_URL=file:./.redwood/test.db
+# TEST_DATABASE_URL=postgresql://gitpod@localhost/test
```

```suggestion
TEST_DATABASE_URL=postgresql://gitpod@localhost/test
```

Uncomment when using Postgres